kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver_componentwise.h

```diff
@@ -254,7 +254,7 @@ class ResidualBasedEliminationBuilderAndSolverComponentwise
                     EquationId[i] = geom[i].GetDof(rVar,pos).EquationId();
 
                 //assemble the elemental contribution
-#ifdef _OPENMP
+#ifdef USE_LOCKS_IN_ASSEMBLY
```

Comment:
this change is NOT aesthetic!

Reply:
This was already blocked, I just saw that it wasn't compiling due to a merge conflict. I only solved that, and you already said that are other things first